### ISSUE TITLE
Leanback/AndroidTV support

### ISSUE THREAD

**Issue by gabriellluz** [x0b/rcx#67]
Hi there.

I currently use Kodi for streaming from OneDrive but every now and then the plugin gets broken and I have to wait a lot until I can use it. If rcx gets proper support for leanback interface (like VLC for Android does), it'll be a lot easier for me.

**Comment by x0b** [comment]
Yeah, a proper TV interface would be nice. +1 to bump this on the priority list.

Have you tried serving the remote with a DLNA serve? You are going to need a ``cache`` if the remote is local, but it should hopefully work.

**Comment by gabriellluz** [comment]
> Yeah, a proper TV interface would be nice. +1 to bump this on the priority list.
> 
> Have you tried serving the remote with a DLNA serve? You are going to need a `cache` if the remote is not local, but it should hopefully work.

nope, did not. You mean, DLNA serving in the same device? Like, I open rcx and use that feature and leave the app open while using a media player on the same device?

**Comment by x0b** [comment]
Yeah, that should work. But with OneDrive, Rclone, RCX and VLC there are quite a few moving parts involved, so some things are not 100% there.

**Comment by gabriellluz** [comment]
OK. I'll try it.
